doc/Security.xml

```diff
@@ -947,13 +995,311 @@ Added certificate-based client authentication</revremark>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>The signature is evaluated by applying ECDSA using secp256r1 and SHA-256 hashing to the base64-encoded header and payload, concatenated by a '.'</para>
+      <programlisting>ECDSA-SHA256 (base64UrlEncode(header) + "." +
+              base64UrlEncode(payload),
+              secp256r1))</programlisting>
+      <para>The evaluated signature is further appended to the encoded header and payload with a '.' separator, to get the final JWT. Appendix <xref linkend="_Ref460769599"/> demonstrates the construction or a JWT.</para>
+      <para>The signature is used to verify the message wasn't changed along the way, and, in the case of tokens signed with a private key, it can also verify the identity of who signed the JWT.</para>
+      <section>
+        <title>Usage of JWT-based client authentication over HTTP</title>
+        <para>Since authentication tokens contain sensitive information that could be easily used to perform replay attacks, JWT-based client authentication shall not be used over unencrypted HTTP connections.</para>
+      </section>
+      <section>
+        <title>Usage of JWT-based client authentication over HTTPS</title>
+        <para>Usage JWT-based client authentication over HTTPS shall adhere to the Authorization Request header Field, as specified by [RFC 6750]. In case of authentication failure, ONVIF-compliant devices shall respond with the error codes specified by [RFC 6750].</para>
+      </section>
+      <section>
+        <title>Usage of JWT-based client authentication over SCTP</title>
+        <para>In scenarios where a SCTP channel is used to exchange commands and responses between the device and the client, it is not possible to embed a JWT within a header. In this case, the JWT can be embedded in the Security Token of the SOAP message, as defined in the WS-Security Basic Security Profile.</para>
+        <para>A client shall use both ValueType and EncodingType. The device shall reject any Binary Security Token not using both <emphasis>ValueType</emphasis> and <emphasis>EncodingType</emphasis>.</para>
+        <para>The EncodingType attribute shall have the value of</para>
+        <programlisting>http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-soap-message-security-1.0#BinarySecurityToken</programlisting>
+        <para>The ValueType attribute shall have the value of</para>
+        <programlisting>urn:ietf:params:oauth:token-type:jwt</programlisting>
+      </section>
+      <section>
+        <title>Usage of JWT-based client authentication over RTSP</title>
+        <para>Since authentication tokens contain sensitive information that could be easily used to perform replay attacks, JWT-based client authentication shall be used with RTSP connections only when HTTPS transport is selected.</para>
```

Comment:
Is this a client or device requirement, if device shall the device reject any HTTP request?